src/router/index.ts

```diff
@@ -56,6 +63,48 @@ const routes: Array<RouteRecordRaw> = [
       depth: 2,
     },
   },
+  {
+    path: "/club/:clubId/awards",
+    name: "Awards",
+    component: AwardsView,
+    meta: {
+      depth: 2,
+    },
+    children: [
```

Comment:
Didn't know about this feature, should we be using it for all the pages within a club, and all the clubs within the whole site?

Reply:
Yeah, we could convert the rest of the routes to use this approach. Would be a cleaner approach for sure. I have created #114 to track this change.